test/parallel/test-url.js

```diff
@@ -125,10 +130,11 @@ var parseTests = {
   },
 
   'http://user%3Apw@www.example.com/' : {
-    'href': 'http://user:pw@www.example.com/',
+    'href': 'http://user%3Apw@www.example.com/',
```

Comment:
This expectation was incorrect previously due to how the module worked before. As per RFC 3986 this would just be the `username` value so it should be percent encoded for the href.